salt/elife-xpub/init.sls

```diff
@@ -225,3 +225,11 @@ elife-xpub-logrotate:
         - template: jinja
         - require:
             - elife-xpub-logs
+
+elife-xpub-vault-credentials-generic:
+    cmd.run:
+        - name: echo {{ salt['vault'].read_secret('secret/projects/elife-xpub/answer') }}
```

Comment:
Needs to be `secret/data/projects/...` for the secret store `KV` version 2:
https://www.vaultproject.io/api/secret/kv/kv-v2.html#read-secret-version